test/flycheck-test.el

```diff
@@ -3507,6 +3507,28 @@ Why not:
    '(4 11 warning "variable 'var2' is never set"
        :id "W221" :checker lua-luacheck)))
 
+(flycheck-ert-def-checker-test lua-luacheck lua nil
```

Comment:
You should replace `nil` by a test name, otherwise the two tests you added end up with the same name (`nil` is replaced by `default`), and only the second one is actually run.